### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by divergentdave**
This updates all NPM dependencies to the latest version. The shrinkwrap file is updated as well, and I ran `npm dedupe` at the end to clean up the node_modules directory a bit.

Going forward, there's a new service at http://greenkeeper.io/ that keeps track of updated dependencies and sends PR requests whenever there's a new release that falls outside of your package's "wanted" range. @konklone interested in signing up?

### CODE REVIEW

File: app/boot.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ var config = require("../config/config"),
 
 module.exports = {
   es: new elasticsearch.Client({
+    apiVersion: "1.7",
```
Comment (konklone) [inline app/boot.js]: Is this tracking the ES client module version, or the ES server version?

**Comment by konklone** [comment]
I signed up for greenkeeper.io's free open source plan, and enabled it for this repo.

I also started documenting the services and accounts used for the project here: https://github.com/konklone/oversight.garden/wiki/Services-used So that it's obvious what we're using and what emails/accounts are tied to them.

File: app/boot.js (hunk 1 above)
Comment (divergentdave) [inline app/boot.js]: My understanding is that the API is defined by the ES server version, so "1.7" here means v1.7 of the server. Version 8.2.0 of the NPM module defaults to the 1.7 API, which is why I pinned it to that before upgrading. The latest client now defaults to targeting the 2.1 API.

More generally, I think we need to straighten out which ES version we're using. I see you mentioned v2.1 when updating the README, should we update to that with the new server, and bump the API the client is using for consistency?

Comment (konklone) [inline app/boot.js]: Hmm, from the [Amazon ES FAQ](https://aws.amazon.com/elasticsearch-service/faqs/):

> Q: Which Elasticsearch version does Amazon Elasticsearch Service support?
> 
> Amazon Elasticsearch Service currently supports Elasticsearch 1.5.2.

:( I was hoping they'd be a little more current than that...

Comment (divergentdave) [inline app/boot.js]: Well the present server is running 1.3.2, so it's compatible enough for our purposes

Comment (konklone) [inline app/boot.js]: ...wow, I didn't realize I'd let it fall so far behind. `*shrug!*`

**Pull request merged**